tests/gui/components/drive-selector.spec.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+const _ = require('lodash');
+const m = require('mochainon');
+const angular = require('angular');
+require('angular-mocks');
+
+describe('Browser: DriveSelector', function() {
+
+  beforeEach(angular.mock.module(
+    require('../../../lib/gui/components/drive-selector/drive-selector')
+  ));
+
+  describe('DriveSelectorController', function() {
+
+    const defaultDrive = {
+      device: '/dev/disk2',
+      name: 'My Drive',
+      size: 123456789,
+      protected: false,
+      system: false
+    };
+
+    let currentDrive;
+
+    let $controller;
+    let $q;
```

Comment:
The `$q` module is not being used anywhere on this test, so we should get rid of it.